source/parser/function.civet

```diff
@@ -573,6 +580,13 @@ function insertReturn(node: ASTNode): void
         insertReturn block
       // NOTE: do not insert a return in the finally block
       return
+    when "PipelineExpression"
+      // At statement level, pipeline generates semicolons between statements
+      // Return the last one
+      semi := exp.children.indexOf ";"
```

Comment:
Should this be `lastIndexOf` or is it always only going to have one?

Reply:
Indeed! Thanks for catching!